app/controllers/tutorials_controller.rb

```diff
@@ -3,8 +3,16 @@ def index
     tutorials
   end
 
+  def show
+    tutorial
+  end
+
   private
 
+  def tutorial
+    @tutorial ||= Tutorial.find(params[:id])
```

Comment:
Routes for tutorials take the form of `tutorials/foo-bar-biz-baz`. By default, Rails assumes the latter part of that URL is an ID (e.g. `/tutorials/1`). FriendlyId is piggy-backing on that through some clever configuration which is why this line uses `params[:id]` instead of something like `params[:slug]`.